internal/nodes/avi_model_nodes.go

```diff
@@ -374,6 +374,7 @@ type AviVsNode struct {
 	Tenant                string
 	ServiceEngineGroup    string
 	ApplicationProfile    string
+	ICAPProfile           string
```

Comment:
Is this field being used somewhere? I am seeing two fields with similar intents.

Reply:
I had been using it to build the reference string, but then dropped that. I'll remove this entry.